src/app/pages/signing/signing.component.html

```diff
@@ -77,9 +109,36 @@ <h5 class="modal-title" id="submit-confirm-modal-title">Are you sure you want to
         You can only submit one message to a friend’s virtrolio per account, so be absolutely certain!
       </div>
       <div class="modal-footer">
-        <button type="button" class="btn btn-primary" routerLink="/msg-sent" data-dismiss="modal">Submit</button>
+        <button type="button" class="btn btn-primary" (click)="createMsg(signingInputArea)" data-dismiss="modal">
+          Send it!</button>
```

Comment:
Button should end on a new line
```suggestion
          Send it!
        </button>
```